wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```
@@ -92,6 +94,25 @@ public static function getSubscribedEvents() {
     ];
   }
 
+  public static function sendInductionEmailToUser(string $op, string $objectName, int $objectId, &$objectRef) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$op'. (undefined)

(UnusedFormalParameter)